migrate/migrate.sh

```diff
@@ -50,25 +89,47 @@ init_options() {
         help_init_options
         exit 1
     fi
+
+    SKIP="${SKIP_POSTGRES} ${SKIP_MONGO} ${SKIP_VAULT}"
 }
 
 init_options $ARGS
 
+
 echo "Note: this script assumes passwordless sudo access on the services box."
 echo "Additionally, the 2.19.x application will be stopped and not started back up."
 
 echo ""
 echo ""
 
-echo "We need some information before we can begin."
-echo "First, let's start with your 2.19.x installation."
-read -p 'Hostname: ' HOSTNAME
-read -p 'SSH Key File: ' KEY_FILE
-read -p 'SSH Username: ' USERNAME
+if [[ -z $HOSTNAME || -z $KEY_FILE || -z $USERNAME ]];
+then
+    echo "We need some information before we can begin."
+    echo "First, let's start with your 2.19.x installation."
+fi
+
+if [ -z $HOSTNAME ];
+then
+    read -p 'Hostname: ' HOSTNAME
+fi
+
+if [ -z $KEY_FILE ];
+then
+    read -p 'SSH Key File: ' KEY_FILE
+fi
+
+if [ -z $USERNAME ];
+then
+    read -p 'SSH Username: ' USERNAME
+fi
+
 HOST="${USERNAME}@${HOSTNAME}"
 
-echo "Now we need the namespace that has CircleCI Server 3.0 installed."
-read -p 'Namespace: ' NAMESPACE
+if [ -z $NAMESPACE ];
+then
+    echo "Now we need the namespace that has CircleCI Server 3.0 installed."
```

Comment:
This needs to be Server 3.1, or maybe just Server 3 ?

Reply:
Ah thanks, been trying to replace it with 3.x but the script was initially written for 3.0.